Makefile

```diff
@@ -1,31 +1,34 @@
 .PHONY : \
 	build \
 	release \
+	release-spm \
 	goldmaster \
 	test \
 	unit-test \
 	integration-test
 
 build: CONFIG = debug
 build: SWIFTBFLAGS = --configuration $(CONFIG)
-build: build-impl
+build: build-impl-spm
 
 clean:
 	rm -rf .build/debug
 	rm -rf .build/release
+	tools/bazelwrapper clean
 
 compiler: release
 
 repo-tools: release
 
-release: CONFIG = release
-release: SWIFTBFLAGS = --configuration $(CONFIG) -Xswiftc -static-stdlib
-release: build-impl
-release:
+release-spm: CONFIG = release
+release-spm: SWIFTBFLAGS = --configuration $(CONFIG) -Xswiftc -static-stdlib
+release-spm: build-impl
+release-spm:
 	ditto .build/$(CONFIG)/Compiler bin/Compiler
 	ditto .build/$(CONFIG)/RepoTools bin/RepoTools
 
-build-impl:
+
+build-impl-spm:
 	swift build $(SWIFTBFLAGS) \
 	    -Xswiftc -target -Xswiftc x86_64-apple-macosx10.13
```

Comment:
Why are we being explicit about the architecture here?

Reply:
How else would you set the target OS in SPM?